pkg/daemon/criruntime/imageruntime/cri.go

```diff
@@ -93,7 +96,18 @@ func (c *commonCRIImageService) PullImage(ctx context.Context, imageName, tag st
 			Annotations: sandboxConfig.Annotations,
 			Labels:      sandboxConfig.Labels,
 		}
+		if pullImageReq.SandboxConfig.Annotations == nil {
+			pullImageReq.SandboxConfig.Annotations = map[string]string{}
+		}
+	} else {
+		pullImageReq.SandboxConfig = &runtimeapi.PodSandboxConfig{
+			Annotations: map[string]string{},
+		}
 	}
+	// Add this default annotation to avoid unexpected panic caused by sandboxConfig is nil
+	// for some poorly thought-out runtime implementations.
```

Comment:
for some poorly thought-out runtime implementations. -> for some  runtime implementations.